core/domain/email_manager.py

```diff
@@ -383,16 +383,14 @@ def _send_bulk_mail(
         '<br>', '\n').replace('<li>', '<li>- ').replace('</p><p>', '</p>\n<p>')
     cleaned_plaintext_body = html_cleaner.strip_html_tags(raw_plaintext_body)
 
-    def _send_bulk_mail_in_transaction(instance_id=None):
+    def _send_bulk_mail_in_transaction(instance_id):
```

Comment:
Edit the docstring for this function listing its arguments.

Reply:
Done!